app/components/common/elements/ContentBox.vue

```diff
@@ -286,12 +302,30 @@ export default {
     margin-top: auto;
   }
 
+  &.no-main-image >:last-child {
+    margin-bottom: auto;
+  }
+
   .middle-text {
     display: flex;
     align-items: center;
     justify-content: space-between;
     width: 100%;
   }
+
+  &__label {
+    @extend %font-14;
+    color: var(--color-primary);
+    padding: 0 12px;
```

Comment:
we should also standardize padding values we use like font. Small, medium, large etc